background/sync-manager.js

```diff
@@ -88,15 +88,12 @@ const syncMan = (() => {
 
     async setDriveOptions(driveName, options) {
       const key = `secure/sync/driveOptions/${driveName}`;
-      await browser.storage.sync.set({
-        [key]: options,
-      });
+      await chromeSync.setValue(key, options);
     },
 
     async getDriveOptions(driveName) {
       const key = `secure/sync/driveOptions/${driveName}`;
-      const r = await browser.storage.sync.get(key);
-      return r[key] || {};
+      return chromeSync.getValue(key) || {};
```

Comment:
This should have `await` in order for `||` to work:
```js
return await chromeSync.getValue(key) || {};
```

Reply:
Good catch. BTW I didn't know `await` has the same precedence as `!` until now.